<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Hospital-based surveillance</title>
  <meta name="description" content="">
  <meta name="author" content="Simon D.W. Frost">
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <link rel="stylesheet" href="libraries/frameworks/revealjs/css/reveal.min.css">
  <link rel="stylesheet" href="libraries/frameworks/revealjs/css/theme/simple.css" id="theme">
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/tomorrow.css" id="theme">
  <!--[if lt IE 9]>
  <script src="lib/js/html5shiv.js"></script>
  <![endif]-->  <link rel="stylesheet" href = "assets/css/ribbons.css">
<link rel="stylesheet" href = "assets/css/simple.css">
<link rel="stylesheet" href = "assets/css/z.css">

</head>
<body>
  <div class="reveal">
    <div class="slides">
      <section class='' data-state='blackout' id='slide-1'>
  
  <h2 style="color:#FFFF00">Hospital based surveillance of norovirus</h2>

<p><br></p>

<h3 style="color:#FFFFFF">Simon Frost, M.A. D.Phil.</h3>

<p><em>Dept. of Veterinary Medicine, and Institute of Public Health</em></p>

<p><em>University of Cambridge</em></p>

<p><img width="10%" style="border:0px;" src="assets/img/cu.svg"></p>

<script src="assets/js/jquery.js"></script>

</section>
<section class='' data-state='cobalt' id='slide-2'>
  
  <h3 style="color:#FFFF00">Acknowledgements</h3>

<ul>
<li>Dept. of Medicine

<ul>
<li>Reidun Lillestol</li>
<li>Lydia Drumright</li>
</ul></li>
<li>Regional Molecular Genetics Laboratory

<ul>
<li>Kim Brugger</li>
</ul></li>
<li>Part II Pathology, <em>DID</em>

<ul>
<li>Chloe Byers</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-3'>
  <h2>Introduction</h2>
  <ul>
<li>Norovirus is an important cause of viral gastroenteritis worldwide</li>
<li>Case reports are increasing, and new strains - often recombinants - emerging

<ul>
<li>How do recombinants emerge, when the infection is acute and associated with aggressive symptoms?</li>
</ul></li>
<li>Outbreaks are seasonal, but no non-human reservoir

<ul>
<li>How does norovirus persist over the summer?</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-4'>
  <h2>Outline:</h2>
  <ul>
<li>Genotyping of norovirus and recombination

<ul>
<li>Old work, in need of updating</li>
</ul></li>
<li>The hospital setting as a crucible for new strains

<ul>
<li>Work in progress</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-5'>
  <h2>Genotyping</h2>
  <ul>
<li>Many viruses are divided into genotypes

<ul>
<li>e.g. Norovirus is divided into several genogroups, genotypes, and clusters

<ul>
<li>GI, GII, GIII, GIV</li>
<li>GII.3, GII.4, etc.</li>
</ul></li>
</ul></li>
</ul>

<p><img src="assets/img/gigiiphylo.png" width="700px" class="centred" style="border: 0px;" /></p>

<div style="font-size:20px">Kroneman et al. Arch. Virol (2013)</div>

</section>
<section class='' data-state='' id='slide-6'>
  <h2>Noronet typing</h2>
  <ul>
<li>Genogroup assignment using BLAST against a reference set of enterovirus genomes</li>
<li>If Norovirus GI or GII, and there is overlap (&gt;100nt) with regions used for genotyping

<ul>
<li>Phylogenetic trees are built using reference alignments of ORF1 and ORF2</li>
<li>Further fine-scale genotyping of GII.4 strains</li>
</ul></li>
<li>Recombination is only detected between ORF1 and ORF2, by a discordant genotype</li>
</ul>

</section>
<section class='' data-state='' id='slide-7'>
  <h2>Issues with Norovirus genotyping</h2>
  <ul>
<li>Recombination may occur within ORFs</li>
<li>Genotype assignment is on the basis of clustering with reference strains

<ul>
<li>What if the sequences cluster with an <em>ancestor</em> of the reference strains?</li>
</ul></li>
<li>What if recombination occurs <em>within</em> genotypes?</li>
<li>Methods assume that the reference alignment is free of recombination

<ul>
<li>What if the reference sequences are themselves recombinants?</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-8'>
  <h2>Subtype Classification using Evolutionary Algorithms</h2>
  <p><img src="assets/img/scueal_method_small.png" width="400px" class="centred" style="border: 0px;" /></p>

<div style="font-size:20px">Kosakovsky Pond et al. PLoS Comp Biol (2009)</div>

</section>
<section class='' data-state='' id='slide-9'>
  <h2>ORF2 II4/II20 recombinants</h2>
  <p><img src="assets/img/gii20gii4_locations.png" width="450px" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-10'>
  <h2>Dynamics of II4/II20 recombinants</h2>
  <p><img src="assets/img/gii20gii4table.png" width="800px" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-11'>
  <h2>Recombination-free references?</h2>
  <ul>
<li>Reference alignments should be free of recombinant variants

<ul>
<li>If present, recombinant sequences should be split up into non-recombinant segments</li>
</ul></li>
<li>Is there recombination in the Noronet reference alignments?

<ul>
<li>Apparently absent from ORF1, as well as GII.4 ORF1 and ORF2 alignments</li>
</ul></li>
<li>Some indication of recombinants in the ORF2 alignment</li>
</ul>

</section>
<section class='' data-state='' id='slide-12'>
  <h2>Bootscan of GII.10</h2>
  <p><img src="assets/img/ii10.png" width="800px" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-13'>
  <h2>Bootscan of GII.20</h2>
  <p><img src="assets/img/ii20.png" width="800px" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-14'>
  <h2>Updates</h2>
  <ul>
<li>The previous analyses conducted on c. 16,000 sequences - now 25,000</li>
<li>Difficult to work out parental strains

<ul>
<li>Use temporal and spatial information to help resolve</li>
</ul></li>
<li>Recombinants are common, while recombination may not be</li>
<li>Noronet reference sets have been updated, and the website often breaks

<ul>
<li>Reverse engineering the typing scheme locally</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-15'>
  <h2>Why so much recombination?</h2>
  <p><img src="assets/img/worobey.jpg" width="400px" class="centred" style="border: 0px;" /></p>

<div style="font-size:20px">Worobey and Holmes J. Gen. Virol. (1999)</div>

</section>
<section class='' data-state='' id='slide-16'>
  <h2>Norovirus and chronic infection</h2>
  <ul>
<li>Norovirus may not always be acute

<ul>
<li>Individuals with immunosuppression may become chronically infected</li>
</ul></li>
<li>Infected individuals may continue to transmit

<ul>
<li>Individuals may not always be isolated when sick</li>
<li>Asymptomatic individuals</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-17'>
  <h2>Modeling chronic infection</h2>
  <p><img src="assets/img/noromodel.png" width="80%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-18'>
  <h2>Recombination, chronic infection, and infectivity</h2>
  <p><img src="assets/img/modelrecomb.png" width="70%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-19'>
  <h2>Multiple infections in hospitals?</h2>
  <ul>
<li>One setting where recombinants may emerge is in hospitals

<ul>
<li>Often have outbreaks, with several introductions of different genotypes</li>
<li>Immunosuppressed individuals may not control the virus, resulting in chronic infection</li>
</ul></li>
<li>May also play a role in persistance over the summer

<ul>
<li>Magnify a small number of cases</li>
<li>Also the potential for asymptomatic healthcare workers to spread infection</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-20'>
  <h2>Brief protocol</h2>
  <ul>
<li>Faecal samples taken prospectively from patients and healthcare workers (HCWs)

<ul>
<li>Acute and chronic patients</li>
<li>Symptomatic and asymptomatic HCWs</li>
</ul></li>
<li>Full genome norovirus sequencing

<ul>
<li>Illumina MiSeq, paired end 2 $\times$ 150bp</li>
<li>Near full length genomes in 48/52 samples to date at 250 $\times$ coverage</li>
</ul></li>
<li>Also testing of samples through TaqMan Low Density Arrays (TLDA)</li>
</ul>

</section>
<section class='' data-state='' id='slide-21'>
  <h2>Norovirus prevalence in submitted samples</h2>
  <p><img src="assets/img/prevalence.png" width="70%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-22'>
  <h2>Sequence processing pipeline</h2>
  <ul>
<li>Snakemake based pipeline

<ul>
<li>Remove duplicate reads (<code>fastuniq</code>)</li>
<li>Trim (<code>Trimmomatic</code>)</li>
<li>Remove human and bacterial reads (<code>deconseq</code>)</li>
<li><em>De novo assembly</em> (<code>iva</code>)</li>
<li>Mapping of reads to contigs (<code>smalt</code>)</li>
<li>Quasispecies assembly (<code>QuasiRecomb</code>)</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-23'>
  <h2>Previous NGS studies of norovirus</h2>
  <ul>
<li>Wong et al. (2013)</li>
<li>Cotten et al. (2014)</li>
<li>Bavelaar et al. (2015)</li>
</ul>

</section>
<section class='' data-state='' id='slide-24'>
  <h2>IVA vs. VICUNA</h2>
  <ul>
<li>Wong et al. (2013) evaluated multiple assemblers (excluding IVA) and concluded that VICUNA gave the best performance</li>
<li>Based on our samples, VICUNA was much faster than IVA but:

<ul>
<li>Contigs often failed to be typed</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-25'>
  <h2>Genotypes of baseline, longest contigs: ORF2</h2>
  <p><img src="assets/img/orf2_tree_nex.png" width="35%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-26'>
  <h2>GII.17</h2>
  <ul>
<li>3 patients were infected with GII.17, an emerging clade of Norovirus first reported in Japan</li>
<li>First report of GII.17 in the UK</li>
</ul>

<p><img src="assets/img/gii17tree.png" width="60%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-27'>
  <h2>GII.4 recombinant</h2>
  <ul>
<li>Wong et al. (2013) reported recombinant viruses, comprising of a GII.4 (New Orleans 2009) ORF1 and a GII.4 (Sydney 2012) ORF2</li>
</ul>

<p><img src="assets/img/wong_recombinants.jpg" width="80%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-28'>
  <h2>Phylogeny of GII.4 recombinant strain</h2>
  <p><img src="assets/img/gii4r_tree.png" width="60%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-29'>
  <h2>Multiple infections</h2>
  <ul>
<li>Two acute patients sampled from different wards, two weeks apart

<ul>
<li>Major: GII.P4 (New Orleans 2009)/GII.4 (Sydney 2012)

<ul>
<li>Minor: GII.17 (in ORF2)</li>
</ul></li>
<li>Major: GII.17

<ul>
<li>Minor: GII.4 (Sydney 2012) in ORF2</li>
</ul></li>
</ul></li>
<li>Chronic patient infected with two different viruses:

<ul>
<li>Major: GII.P4 (New Orleans 2009)/GII.4 (Sydney 2012)

<ul>
<li>Minor: SaV I (Sapovirus GI.1)</li>
</ul></li>
</ul></li>
<li>Chronic patient, sampled from home

<ul>
<li>Major: GII.P4 (New Orleans 2009)/GII.4 (Sydney 2012)

<ul>
<li>Minor: GII.2 (in ORF2)</li>
</ul></li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-30'>
  <h2>Coverage in chronic patient</h2>
  <p><img src="assets/img/coverage.png" width="70%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-31'>
  <h2>Longitudinal changes in chronic patient</h2>
  <p><img src="assets/img/alltimes.png" width="60%" class="centred" style="border: 0px;" /></p>

</section>
<section class='' data-state='' id='slide-32'>
  <h2>Conclusions</h2>
  <ul>
<li>The hospital setting is a highly dynamic environment of norovirus transmission</li>
<li>Multiple introductions with multiple subtypes</li>
<li>Multiple infections</li>
<li>Chronic infections</li>
<li>Recombination?</li>
</ul>

</section>
<section class='' data-state='' id='slide-33'>
  <h2>Ongoing work</h2>
  <ul>
<li>Complete GII.17 outbreak</li>
<li>More chronic patients</li>
<li>Healthcare workers</li>
<li>Methodology for mixed infections

<ul>
<li>Detection and genotyping</li>
<li>Recombination</li>
<li>Linkage to epidemiology</li>
</ul></li>
</ul>

</section>
<section class='' data-state='yellow' id='slide-34'>
  
  <h2 style="color:#000000">Thanks!</h1>

<br>

<h2>
<span style="font-size:24px"><img src="assets/img/gmail.png" style="background:none; border:none; box-shadow:none; vertical-align:middle"/> &nbsp;<a href="mailto:sdwfrost@gmail.com" target="_blank" style="vertical-align:middle">sdwfrost@gmail.com</a></span>
<span style="font-size:24px"><img src="assets/img/twitter-icon.png" style="background:none; border:none; box-shadow:none; vertical-align:middle"/> &nbsp;<a href="http://www.twitter.com/sdwfrost" target="_blank" style="vertical-align:middle">@sdwfrost</a></span>
<span style="font-size:24px"><img src="assets/img/GitHub-Mark/PNG/GitHub-Mark-32px.png" style="background:none; border:none; box-shadow:none; vertical-align:middle"/> &nbsp;<a href="http://github.com/sdwfrost" target="_blank" style="vertical-align:middle">http://github.com/sdwfrost</a></span>
</h2>

</section>
    </div>
  </div>
</body>
  <script src="libraries/frameworks/revealjs/lib/js/head.min.js"></script>
  <script src="libraries/frameworks/revealjs/js/reveal.min.js"></script>
  <script>
  // Full list of configuration options available here:
  // https://github.com/hakimel/reveal.js#configuration
  Reveal.initialize({
    controls: true,
    progress: true,
    history: true,
    center: false,
    theme: Reveal.getQueryHash().theme || 'simple', 
    transition: Reveal.getQueryHash().transition || 'linear', 
    dependencies: [
    // Cross-browser shim that fully implements classList -
    // https://github.com/eligrey/classList.js/
      { src: 'libraries/frameworks/revealjs/lib/js/classList.js', condition: function() { return !document.body.classList;}},
      // Zoom in and out with Alt+click
      { src: 'libraries/frameworks/revealjs/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
      // Speaker notes
      { src: 'libraries/frameworks/revealjs/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
      // Remote control your reveal.js presentation using a touch device
      //{ src: 'libraries/frameworks/revealjs/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } }
      ]
  });
  </script>  <!-- MathJax: Fall back to local if CDN offline but local image fonts are not supported (saves >100MB) -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      processEscapes: true
    }
  });
</script>
<script type="text/javascript" src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
<!-- <script src="https://c328740.ssl.cf1.rackcdn.com/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script> -->
<script>window.MathJax || document.write('<script type="text/x-mathjax-config">MathJax.Hub.Config({"HTML-CSS":{imageFont:null}});<\/script><script src="libraries/widgets/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><\/script>')
</script>
<!-- LOAD HIGHLIGHTER JS FILES -->
<script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
 

</html>